app/jobs/result_monitor_load.rb

```diff
@@ -0,0 +1,19 @@
+# Load a result from S3 into the db
+require 'logger'
+class ResultMonitorLoad
```

Comment:
nit: maybe ResultMonitorLoader to be the actual object name

Reply:
ok